packages/switch/src/Switch.types.ts

```diff
@@ -0,0 +1,10 @@
+export interface SwitchProps {
+  /**
+   * If true, the switch is displayed as checked
+   */
+  checked?: boolean;
+  /**
+   * Function that will be called every time the Switch value is changed. The argument `isChecked` represent the new/target value of the `checked` attribute.
+   */
+  onChange?: (isChecked: boolean) => void;
```

Comment:
what about `disabled` state and the `className` prop ?

Reply:
`className` prop added. `disabled` state will add in a separate PR https://heycar.atlassian.net/browse/HEYUI-119

Reply:
Actually, added `disabled` state to this PR as well